test/e2e/scheduling/nvidia-gpus.go

```diff
@@ -182,6 +183,14 @@ func testNvidiaGPUs(f *framework.Framework) {
 	framework.ExpectNoError(err, "getting resource usage summary")
 }
 
+func logContainers(f *framework.Framework, pod *v1.Pod) {
+	for _, container := range pod.Spec.Containers {
+		logs, err := framework.GetPodLogs(f.ClientSet, f.Namespace.Name, pod.Name, container.Name)
+		framework.ExpectNoError(err, "Should be able to get container logs")
```

Comment:
better to output `container.Name` also into the error message because this is used for multiple containers and it could be helpful if we face an issue.

Reply:
Thanks. That makes sense. Added the container name to the error message.